cockroachcloud/export-metrics.md

```diff
@@ -94,25 +94,26 @@ Perform the following steps to enable metrics export from your {{ site.data.prod
         "Version": "2012-10-17",
         "Statement": [
             {
+                "Action": [
+                    "logs:CreateLogGroup",
+                    "logs:CreateLogStream",
+                    "logs:DescribeLogGroups",
+                    "logs:DescribeLogStreams",
+                    "logs:PutRetentionPolicy",
+                    "logs:PutLogEvents"
+                ],
                 "Effect": "Allow",
-                "Principal": {
-                    "AWS": "arn:aws:iam::{your_aws_acct_id}:root"
-                },
-                "Action": "sts:AssumeRole"
-            },
-            {
-                "Effect": "Allow",
-                "Principal": {
-                    "AWS": "arn:aws:iam::{your_aws_acct_id}:root"
-                },
-                "Action": "sts:AssumeRole"
+                "Resource": [
+                    "arn:aws:logs:*:{your_aws_acct_id}:log-group:{log_group_name}:*"
```

Comment:
I noticed that I received errors with this resource name.

We were able to fix by using this instead:
```
"Resource": [
    "arn:aws:logs:*:{your_aws_acct_id}:log-group:{log_group_name}",
    "arn:aws:logs:*:{your_aws_acct_id}:log-group:{log_group_name}:log-stream:*"
]
```

Once we swapped to this, it worked 😎